libmozevent/phabricator.py

```diff
@@ -21,10 +23,16 @@ class PhabricatorBuild(object):
     """
 
     def __init__(self, request):
-        self.diff_id = int(request.rel_url.query.get("diff", 0))
-        self.repo_phid = request.rel_url.query.get("repo")
-        self.revision_id = int(request.rel_url.query.get("revision", 0))
-        self.target_phid = request.rel_url.query.get("target")
```

Comment:
I think these are not actually requests, the actual requests are done in the phabricator library in libmozdata